pyboy/plugins/game_wrapper_tetris.py

```diff
@@ -150,7 +152,7 @@ def reset_game(self, timer_div=None):
         """
         After calling `start_game`, you can call this method at any time to reset the game.
 
-        Args:
+        Kwargs:
```

Comment:
I might have to pass the kwargs of openai_gym to reset_game too then

Reply:
Should that be done before merging?

Reply:
Yeah maybe, we need to test that this works well also (maybe on tetris with a fixed seed)

Reply:
Would you do that?

Reply:
I don't have the time this week sadly :'(